android/build/filament-tasks.gradle

```diff
@@ -64,8 +64,8 @@ class MaterialCompiler extends DefaultTask {
     File matcPath
 
     @SuppressWarnings("GroovyUnusedDeclaration")
-    @InputDirectory
-    File inputDir
+    @InputFile
```

Comment:
Are we sure about this change?

Reply:
Restored this, replaced with a different fix.